content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:8800 check
+   server app-direct-node02 10.240.0.39:8800 check
+   server app-direct-node03 10.240.0.71:8800 check
+###########################################################
+frontend kube-api-frontend
+   mode tcp
+   bind    *:6443
+   default_backend kube-api-backend
+backend kube-api-backend
+   mode                    tcp
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:6443 check
+   server app-direct-node02 10.240.0.39:6443 check
+   server app-direct-node03 10.240.0.71:6443 check
+
+```
+
+For more details on the structure of the config file, please refer to the HA Proxy [documentation](https://www.haproxy.com/blog/the-four-essential-sections-of-an-haproxy-configuration/)
+
+To copy the config file into the VM, we are going to use scp. The command below will copy it to your home folder on the VM.
+
+```shell
+gcloud compute scp haproxy.cfg app-direct-ha-proxy:~/
+```
+
+Now we install Ha Proxy using apt. 
+
+
+```shell
+sudo apt-get update
+sudo apt install -y haproxy
+```
+
+The install will create a default config file. We will back it up an then move ours to the /etc/haproxy directory created by the install.
+
+```shell
+sudo mv /etc/haproxy/haproxy.cfg /etc/haproxy/haproxy.bak
+sudo mv ~/haproxy.cfg /etc/haproxy/haproxy.cfg
+```
+
+Last step is to restart Ha Proxy
+
+```shell
+sudo systemctl restart haproxy
+```
+
+To validate the install, browse to http://<public-ip-of-vm>:8080. Below is a screenshot of what it should look like. The dashboard at this point does not contain much valuable information as all the frontend/backend services we configured are not running yet.
+
+![haproxy](/images/guides/kots/ha-cluster-ha-proxy.png)
+
+### Run the Kots Installer & Deploy Application
+
+The next step is to install Kots on the first node, in our case, app-direct-node-01. It's important to note that in reality, you could run the installer on any of the three nodes.
+
+To run the installer, ssh into the node and run the 'one line installer'. You can get this command from the Vendor portal as shown below.
+
+![Channels](/images/guides/kots/ha-cluster-portal-channels.png)
+
+However, in order to enable High Availability we are going to add the '-s ha' [advanced option](https://kurl.sh/docs/install-with-kurl/advanced-options)
+
+```shell
+curl -sSL https://k8s.kurl.sh/appdirect-unstable | sudo bash -s ha
+```
+
+When you include the '-s ha' option, you will be promoted right away for the load balancer address. As shown in the figure below, we are providing the 'internal' IP address. This address is used by the Kubernetes services in the nodes to talk to each other. Using a public IP addresses addds extra layers that are not needed. 
+
+This process will take several minutes as it will install Kubernetes as well as the Application Administration toolset. Once it is finished, you should see something similar as shown below:
```

Comment:
is there a reason why `Application Administration` is title case?

Reply:
I guess I was thinking first of "will install the Application Admin console", but then added the tooling part, which I guess makes it 'a set of things' versus 'one specific item I am calling by its proper name"... so should be lower case

Reply:
I changed this to:

This process will take several minutes as it will install a Kubernetes 'stack' that will include the add-ons defined in the Kubernetes Installer, including Kots. Once it is finished, you should see something similar as shown below: